### PR TITLE
Add metagenomeSeq normalization tool: Cumulative Sum Scaling.

### PR DESCRIPTION
**Pull request by blankenberg**

### CODE REVIEW

File: tools/metagenomeseq/metagenomeseq_normalization.xml [hunk 1]
```diff
@@ -0,0 +1,148 @@
+<tool id="metagenomeseq_normalizaton" name="metagenomeSeq Normalization" version="1.16.0-0.0.1">
+    <description>Cumulative sum scaling</description>
+    <requirements>
+        <requirement type="package" version="1.16.0">bioconductor-metagenomeseq</requirement>
+        <requirement type="package" version="1.2.0">bioconductor-biomformat</requirement>
+    </requirements>
+    <stdio>
```
Comment (bgruening) [inline tools/metagenomeseq/metagenomeseq_normalization.xml]: Can be replaced by `<command detect_errors="exit_code">`

File: tools/metagenomeseq/metagenomeseq_normalization.xml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        </when>
+    </conditional>
+
+    <param name="log2" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="False" label="Log2 transform scaling" help=""/>
+    <param name="sl" type="integer" value="1000" label="The value to scale by." help="(default=1000)"/>
+
+    <param name="include_outputs" type="select" multiple="True" label="Datasets to create">
+        <option value="output_tabular" selected="True">Normalized Tabular Matrix</option>
+        <!-- <option value="output_biom" selected="True">Normalized BIOM</option> -->
+        <option value="output_stats" selected="True">Simple Statistics</option>
```
Comment (bgruening) [inline tools/metagenomeseq/metagenomeseq_normalization.xml]: `true` can also be written lower case, like the `false` below.

File: tools/metagenomeseq/metagenomeseq_normalization.xml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+::
+  
+  Requires a BIOM formatted file for input. 
+
+Outputs
+-------
+
+::
+  
+  Creates a normalized and scaled output abundance matrix in Tabular or BIOM (broken: https://github.com/joey711/biomformat/issues/4) format. Additionally can create simple statistics and the RScript.
```
Comment (bgruening) [inline tools/metagenomeseq/metagenomeseq_normalization.xml]: Broken BIOM format? This seems to be confusing for the user? What is the consequence of this bug?

Comment (bgruening) [inline tools/metagenomeseq/metagenomeseq_normalization.xml]: BIOM is also not an output anymore, it is commented out.

Comment (blankenberg) [inline tools/metagenomeseq/metagenomeseq_normalization.xml]: Will update help, thanks for catching. 

Unfortunately, the bioconductor biomformat package writes out something that almost looks like BIOM, but isn't BIOM (see https://github.com/joey711/biomformat/issues/4). Wonder how many other R packages are unfortunately relying on this broken behavior. Reading BIOM does seem to work, however.

**Comment by bgruening** [comment]
Alle guten Dinge sind 3!

**Pull request merged**